drivers/display/display_st7789v.c

```diff
@@ -0,0 +1,368 @@
+/*
+ * Copyright (c) 2017 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ * Copyright (c) 2019 Marc Reilly
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "display_st7789v.h"
+
+#include <device.h>
+#include <spi.h>
+#include <gpio.h>
+#include <misc/byteorder.h>
+#include <display.h>
```

Comment:
Please update the include paths to reflect changes in #16540

Reply:
done

Reply:
spi, gpio, and display paths also need to be updated